README.md

```diff
@@ -1,8 +1,8 @@
 # KFServing
-KFServing provides a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and custom containers. KFServing brings cutting edge serving features like Scale to Zero and Canary Rollouts to your ML deployments.
+KFServing provides a Kubernetes [Custom Resource Definition](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) for serving ML Models on arbitrary frameworks. It aims to solve 80% of model serving use cases by providing performant, high abstraction interfaces for common ML frameworks like Tensorflow, XGBoost, ScikitLearn, PyTorch, and custom containers. KFServing brings cutting edge serving features like GPU Autoscaling, Scale to Zero, and Canary Rollouts to your ML deployments.
```

Comment:
GPU autoscaling, because knative?

Reply:
Yeah. It's quite difficult to effectively autoscale GPUs on GPU metrics. Different workloads max out GPUs at different metric values, so it's easier to just use the queue which autoscales generically.